composer.json

```diff
@@ -24,7 +24,7 @@
         "drupal/phingdrushtask": "dev-7.x-2.x#13428ee7a229da3b7e6be4687587048150b1fd25",
         "drupal/r4032login": "dev-1.x#4b2077aa70e3f7b00b8a9cba25af5b948ba2e3b9",
         "drupal/search_api": "1.0-alpha17",
-        "drupal/search_api_solr": "~1.0-alpha5",
+        "drupal/search_api_solr": "1.0-alpha6",
```

Comment:
Had to lock `search_api_solr` because the new beta1 release is failing. This is being handled in ISAICP-2875.